util/src/main/i18n/templates/gamemode.properties

```diff
@@ -208,8 +208,9 @@ flag.captureDenied = {0} will be captured when allowed
 flag.return = {0} has been returned
 
 # {0} = flag name
-# {1} = duration (e.g. "5 seconds")
-flag.willRespawn = {0} will respawn in {1}
+# {1} = " to " + flag post name. If post name is null, {1} = ""
+# {2} = duration (e.g. "5 seconds")
+flag.willRespawn = {0} will respawn{1} in {2}
```

Comment:
The "to" will need to be translated as well, so I would separate this into 2 translation strings. One where post name is null and another where it is not.